cheddar/cheddar-integration-aws/build.gradle

```diff
@@ -8,7 +8,14 @@ dependencies {
     compile project(':cheddar:cheddar-messaging')
     compile project(':cheddar:cheddar-persistence')
 
-    compile 'com.amazonaws:aws-java-sdk:1.8.11'
+    compile 'com.amazonaws:aws-java-sdk-s3:1.9.23'
```

Comment:
@clicktravel-basharat - Amazon have changed so that the SDK is now modular meaning there are separate jar files for each offering they have. We could have just used "compile 'com.amazonaws:aws-java-sdk:1.9.23'" however this pulls in 45 AWS jar files plus their 3rd party dependancies so this is an attempt to only use what we need with the upgrade.